extensions/amp-story/amp-story.md

```diff
@@ -1136,6 +1135,42 @@ You can apply multiple entrance animations on one element (for example, an eleme
 If a composed animation is supposed to start after the end of a separate element's animation, make sure that all nested elements that compose the animation have the attribute `animate-in-after` set to the same `id`.
 {% endcall %}
 
+## Optional customization
+
+### `data-text-background-color`
+
+Use this attribute inside a `amp-story-page` to highlight the text with a color of your choice. To highlight the entire block, add this attribute directly to the text element. To only highlight the text, add the attribute and your text to an inner `<span>`.
```

Comment:
Change to:
The `data-text-background-color` attribute highlights the text of the element with a specified color.  To highlight the entire block, add this attribute directly to the text element. To only highlight the text, add the attribute and text to an inner `<span>`.

Reply:
Done!